packages/grafana-ui/src/components/TimePicker/TimePickerContent/TimePickerCalendar.tsx

```diff
@@ -248,9 +251,15 @@ const Header = memo<Props>(({ onClose }) => {
 });
 
 const Body = memo<Props>(props => {
```

Comment:
Maybe just destruct this in the func declaration instead of several places below?
```suggestion
const Body = memo<Props>(({from, to, onChange}) => {
```

Reply:
great feedback!